images/clusterloader2/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.14.4 AS clusterloader-builder
+RUN go version
+WORKDIR /go/src/k8s.io
+RUN git clone https://github.com/njtran/perf-tests
+WORKDIR ./perf-tests/clusterloader2
+RUN mkdir bin
```

Comment:
No reason to create bin, we're copying it out of there anyways.

Reply:
Good point